src/main/kotlin/no/nav/klage/oppgave/clients/OppgaveClient.kt

```diff
@@ -133,10 +134,13 @@ class OppgaveClient(
             .block() ?: throw OppgaveNotFoundException("Oppgave could not be fetched")
     }
 
-    private fun logTiming(function: () -> OppgaveResponse): OppgaveResponse {
+    private fun logTimingAndWebClientResponseException(function: () -> OppgaveResponse): OppgaveResponse {
         val start: Long = currentTimeMillis()
         try {
             return function.invoke()
+        } catch(ex: WebClientResponseException) {
+            logger.error("Got a {} error calling Oppgave {} {} with message {}", ex.statusCode, ex.request.method, ex.request.uri, ex.responseBodyAsString)
```

Comment:
burde det ikke vært norsk?

Reply:
All logging er på engelsk mener jeg?

Reply:
Jeg prøver egentlig bare å tilpasse meg teamet jeg altså.. Jeg ville normalt logget dette på norsk, men trodde teamet hadde bestemt seg for engelsk. :) 

Vi har vel sagt det før, men kanskje det faktisk er på tide nå med en team-prat om språkbruk.. ;)

Reply:
Dere får vurdere denne PRen uten tanke på språk, så får vi ta diskusjonen på det senere!